Automattic/newspack-network
# feat(content-distribution): posts column

Implements a column in the posts list for the posts' content distribution status.

<img width="402" alt="image" src="https://github.com/user-attachments/assets/7cb16757-5fb7-412a-8a54-f463a7525e41" />

<img width="335" alt="image" src="https://github.com/user-attachments/assets/8b8166f3-eac4-46b8-aadd-870b99522455" />

<img width="318" alt="image" src="https://github.com/user-attachments/assets/9d0b912e-40b6-40ae-bc8b-534a2f5e47ef" />

### Testing

1. Distribute a post and visit the posts list
2. Confirm the post has the ☁️ ⬆️  icon and the title and screen reader text match the number of sites it's been distributed to
3. In the receiving node, confirm the 🔗 icon is presented in the list with proper text
4. Click the link and confirm it redirects you to the original post
5. Use the shell to unlink the post in the node:
```php
( new Incoming_Post( <post-id> ) )->set_unlinked();
```
6. Refresh the node's post lists and confirm the icon and texts now represent the unlinked state
